jena-tdb2/src/main/java/org/apache/jena/tdb2/store/DatasetGraphTDB.java

```diff
@@ -157,4 +178,61 @@ public NodeTupleTable chooseNodeTupleTable(Node graphNode) {
             // Includes Node.ANY and union graph
             return getQuadTable().getNodeTupleTable();
     }
+
+    private static class DistinctKeyPrefixIterator implements Iterator<Node> {
```

Comment:
leftover?

Reply:
Yes this was a first pass implementation before I went with the low level iterator

- [ ] Remove discarded implementation prototype